.expeditor/scripts/release_habitat/build_docker_image.ps1

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env powershell
+
+#Requires -Version 5
+
+param (
+  # The builder channel to install packages from. Defaults to unstable
+  [string]$ReleaseChannel="unstable",
+  # The docker image name. Defaults to "habitat/win-studio-x86_64-windows
+  [string]$imageName = "habitat/win-studio-x86_64-windows",
+  # the tag to use for the base image
+  [string]$BaseTag,
+  # the URL for the Builder instance from which to pull packages
+  [string]$BldrUrl = "https://bldr.habitat.sh"
```

Comment:
should this be prod?

Reply:
It's the default value, which we override... seems like the logical choice for a default, no?